app/qml/ExternalResourceBundle.qml

```diff
@@ -185,17 +186,21 @@ Item {
         }
     }
 
-    MessageDialog {
-        id: pickerNotImplementedInfo
-        visible: false
-        title: qsTr( "Not implemented" )
-        text: qsTr( "Picker from gallery on iOS is not supported yet." )
-        icon: StandardIcon.Warning
-        standardButtons: StandardButton.Cancel
-        onRejected: {
-           visible = false
+    ImagePicker {
+      id: picker
```

Comment:
please call it iospicker and IOS imagepicker since it is not for any other platform.